wiki/Building-From-Source.md

```diff
@@ -65,14 +80,41 @@ If you want to edit and rebuild the code, you can skip the `prepare-deps` comman
 For the client you need install:
 
 * [Android Studio](https://developer.android.com/studio) or the [sdkmanager](https://developer.android.com/studio/command-line/sdkmanager)
-* Platform Tools 33 (Android 13)
-* Latest NDK (currently v25.1.8937393)
-* Set the environment variable `JAVA_HOME`
-  * For example on Windows: `C:\Program Files\Android\Android Studio\jre`
-* Set the environment variable `ANDROID_SDK_ROOT`
-  * For example on Windows: `%LOCALAPPDATA%\Android\Sdk`
-* Set the environment variable `ANDROID_NDK_HOME`
-  * For example on Windows: `%LOCALAPPDATA%\Android\Sdk\ndk\25.1.8937393`
+* Latest Android SDK Platform-Tools 33 (Android 13)
+* Latest Android NDK (currently v25.1.8937393)
+
+On Linux, the three mentioned developer applications can be installed on Linux from upstream.
+
+* **Arch**
+
+  ```bash
+  sudo pacman -S tandroid-sdk-platform-tools sdkmanager android-ndk
+  ```
+
+* **Gentoo**
+
+  * `dev-util/android-studio`
+  * `dev-util/android-sdk-update-manager`
+  * `dev-util/android-ndk >= 25.1`
+
+* **Debian 12 / Ubuntu 22.10 / Pop!\_OS 22.10** (Debian requires to have the `non-free` repository enabled)
+  ```bash
+  sudo apt install android-sdk-platform-tools-common sdkmanager google-android-ndk-r25b-installer
+  ```
```

Comment:
There have been reports that the Arch package doesn't work. I personally prefer leaving the SDK/NDK installation instructions as unspecified, since it varies greatly from distro to distro. Instructions can be easily found online.

Reply:
Would it better to link to their wiki page, then? for all three distros, or just Arch? Gentoo too?

Reply:
IIRC using packages is more difficult because then the sdkmanager (and by extension the build system) can't automatically install the required SDK components, so you need to make sure to have them manually (either by installing everything from the repos or running sdkmanager as root)

Reply:
I think ideally we should just remove the old client asap and just switch to the one which doesn't require android studio stuff to build

Reply:
So it'd be best to leave out the mention of these packages?